STYLE.md

```diff
@@ -0,0 +1,270 @@
+#Telia Company Terraform Style Guide
+
+## Introduction
+
+This guide gives developers a guidance in the coding conventions used by Telia Company  for Terraform's HashiCorp Configuration Language (HCL). Terraform allows infrastructure to be described as code, which in turns makes changes and implementations of readable in a structured way. As such, we should adhere to a style guide to ensure readable and high quality code. As Terraform utilises [HCL](https://github.com/hashicorp/hcl), you may wish to take a detailed look at its [syntax guide](https://github.com/hashicorp/hcl/blob/master/README.md#syntax) for further guidance.
+
+Inspired by [The Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) and [The Puppet Style Guide](https://docs.puppetlabs.com/guides/style_g uide.html).
+
+## Syntax
+
+- Strings are in double-quotes.
```

Comment:
I don't know how it will be in HCL2 but the current terraform docs recommends [booleans](https://www.terraform.io/docs/configuration/variables.html#booleans) to be quoted.
```
It is recommended for now to specify boolean values for variables as the 
strings "true" and "false", to avoid some caveats in the conversion process.
A future version of Terraform will properly support boolean values and so
relying on the current behavior could result in backwards-incompatibilities
at that time.
```